include/nrf_cloud.h

```diff
@@ -59,8 +59,17 @@ enum nrf_cloud_sensor {
 	NRF_CLOUD_SENSOR_GPS,
 	/** The FLIP movement sensor on the device. */
 	NRF_CLOUD_SENSOR_FLIP,
+	/** The Button press on the device. */
```

Comment:
It seems previoud convention has been to comment this with the device string defined in `nrf_cloud_codec.c`, so it could be considered to change this (and the following) to something like
`/** The BUTTON sensor on the device. */`